index.d.ts

```diff
@@ -0,0 +1,301 @@
+export interface CSSProperties {
+  /**
+   * In dev mode, adding a `label` string prop will reflect its value in devtools. Useful
+   * when debugging, and a good alternative to 'semantic' classnames.
+   */
+  label?: string;
+  // for now just allow everything
+  [propertyName: string]: any;
```

Comment:
I hope we can re-use some standard interface in the future here. See https://github.com/Microsoft/TypeScript/issues/10820.